starters/gatsby-wordpress-starter/__tests__/snapshotTests/pagesIndex.test.jsx

```diff
@@ -2,28 +2,20 @@ import * as Gatsby from 'gatsby'
 import { render } from '@testing-library/react'
 
 import PageIndexTemplate from '../../src/templates/pagesIndex'
-import data from '../data/pages.json'
-
-const StaticQuery = vi.spyOn(Gatsby, 'StaticQuery')
+import pages from '../data/pages.json'
 
 describe('<PageIndexTemplate />', () => {
 	beforeEach(() => {
-		StaticQuery.mockImplementationOnce(() => {
```

Comment:
Curios as to why we no longer need the StaticQuery here?

Reply:
The mock is not needed because we are no longer using  a `pageQuery` in the component and instead pass data via `pageContext` from `gatsby-node.js`.

Reply:
My understanding of these mocks is they are mocking the queries in gatsby-node > `createPages`

Reply:
Ah, I could have misunderstood their purpose but I only seen that kind of mock used in their [docs](https://www.gatsbyjs.com/docs/how-to/testing/testing-components-with-graphql/) when testing page queries. The mocks were necessary before but the tests do pass fine without them since I removed the queries from the components themselves. I can add them back if we want but I'm not sure what they are doing without the use of page query.

Reply:
hmm well maybe we don't need them anywhere then? Are you able to remove it from it from `pagination.test.jsx` as well?

Reply:
Yes and done! I removed the mock from `pagination.test.jsx` in this most recent update. We still need mocks in the individual `page` and `post` tests since those are using page queries to get a unique post based off of an id.